cmd/kube-controller-manager/app/mizarcontrollers.go

```diff
@@ -154,8 +180,20 @@ func startArktosNetworkController(ctx *ControllerContext, grpcHost string, grpcA
 	svcKubeClient := clientset.NewForConfigOrDie(netKubeconfigs)
 	informerFactory := externalversions.NewSharedInformerFactory(networkClient, 10*time.Minute)
 
+	// Used to create CRDs - VPC or Subnet of tenant
+	dynamicClient := dynamic.NewForConfigOrDie(netKubeconfigs)
+
+	// Used to create mapping to find out GVR via GVK
+	discoveryClient, err := discovery.NewDiscoveryClientForConfig(netKubeconfigs)
```

Comment:
is this discovery client a must have?

Reply:
yes. it is used to find out the mapping between GVK(Group Version Kind) and GVR(Group Version Resource).